### DISCUSSION TITLE
What react framework is used for the tailwindcss doc site?

### DISCUSSION THREAD

**Discussion by mcheck** [tailwindlabs/tailwindcss#2783] [General]
Using the tailwind doc site is a breeze and pleasure. The Algolia search is excellent.
Wondering what react framework is used to build the doc site?

Thanks very much!

**Comment by axpro** [comment]
I guess, looking at the source, that it uses [Next.js](https://nextjs.org/).

**Reply by mcheck** [reply]
I looked at Wappalyzer which said react. I didn’t look too far into the code, though.

**Comment by erhankaradeniz** [comment]
It’s using Next.js, you can see that here: [https://github.com/tailwindlabs/tailwindcss.com](https://github.com/tailwindlabs/tailwindcss.com)

**Comment by mcheck** [comment]
Thanks all.
